nfsstat/tests/compose/compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+  nfs-server:
+    image: gists/nfs-server
+    container_name: nfs-server
+    privileged: true
+    ports:
+      - 2049:2049
```

Comment:
Is exposing the port to the host required given that it is used by `nfs-client` which runs within the same Compose network?

Reply:
It may not be required indeed, I will check